person/src/com/person/core/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.person.core.exceptions;
+
+public class NotFoundException extends ValidationException {
```

Comment:
@tatevikb also you can create a constructor which will get an exception as well and with using that one you will have detailed explanation